### PR TITLE
[ID] Update `Medals`

### PR DESCRIPTION
**Pull request by ZeroPyrozen**

### CODE REVIEW

File: wiki/Medals/id.md [hunk 1]
```diff
@@ -1,16 +1,59 @@
----
-outdated: true
----
-# Pencapaian
+# Medali
 
-**Pencapaian** (Achievements) adalah sebuah badge kecil di bagian bawah profil para user yang mengindentifikasi seseorang telah mencapai sebuah target yang telah ditentukan. Saat ini, badge dibagi dalam 5 grup
+**Medali**, sebelumnya dikenal sebagai *achievement*/pencapaian, adalah lencana-lencana pada profil user yang mengindikasikan target tertentu telah dicapai oleh user. Saat ini, ada **247** medali yang dapat dibuka dalam delapan kategori secara keseluruhan yaitu: [Beatmap Packs](#beatmap-packs), [Skill](#skill), [Dedication](#dedication), [Mod Introduction](#mod-introduction), [Hush-Hush](#hush-hush), [Beatmap Spotlights](#beatmap-spotlights), [Seasonal Spotlights](#seasonal-spotlights), dan [Beatmap Challenge Packs](#beatmap-challenge-packs).
```
Comment (mousesalt) [inline wiki/Medals/id.md:3]:
```suggestion
**Medali**, sebelumnya dikenal sebagai *achievement*/pencapaian, adalah lencana-lencana yang diletakkan pada profil pengguna untuk mengindikasikan target tertentu yang telah dicapai oleh pengguna. Saat ini, ada **247** medali yang dapat dibuka dalam delapan kategori yang berbeda secara keseluruhan, yaitu: [Beatmap Packs](#beatmap-packs), [Skill](#skill), [Dedication](#dedication), [Mod Introduction](#mod-introduction), [Hush-Hush](#hush-hush), [Beatmap Spotlights](#beatmap-spotlights), [Seasonal Spotlights](#seasonal-spotlights), dan [Beatmap Challenge Packs](#beatmap-challenge-packs).
```

Comment (mousesalt) [reply wiki/Medals/id.md:3]: fixed confusing rewording

File: wiki/Medals/id.md [hunk 2]
```diff
@@ -1,16 +1,59 @@
...
+
+Namun perlu diperhatikan bahwa konten-konten dari seluruh Video Game Packs, Anime Packs, Internet! Packs, dan Rhythm Game Packs telah berubah drastis sejak [pengenalan pack-pack tersebut pada 2008](https://osu.ppy.sh/community/forums/topics/1853). Untuk daftar beatmap-beatmap packs ini pada kondisi aslinya, lihat [Legacy Beatmap Packs](/wiki/Medals/Legacy_Beatmap_Packs).
+
+## Skill
+
+**Medali Skill** adalah medali yang diberikan ketika pemain meraih pencapain berbasis kemampuan, baik dengan meraih jumlah [kombo](/wiki/Beatmapping/Combo) tertentu, atau meraih peringkat tertentu pada [peringkat poin performa global](https://osu.ppy.sh/rankings/osu/performance), atau dengan mencapai penyelesaian atau [full combo](/wiki/Full_combo) pada beatmap dengan status [Qualified](/wiki/Beatmap/Category#qualified), [Ranked](/wiki/Beatmap/Category#ranked), atau [Approved](/wiki/Beatmap/Category#approved) pada rentang [star rating](/wiki/Beatmap/Difficulty#star-rating) tertentu.
```
Comment (mousesalt) [inline wiki/Medals/id.md:15]:
```suggestion
**Medali Skill** adalah medali yang diberikan ketika pemain meraih pencapaian berbasis kemampuan, baik dengan meraih jumlah [kombo](/wiki/Beatmapping/Combo) tertentu, atau meraih peringkat tertentu pada [peringkat poin performa global](https://osu.ppy.sh/rankings/osu/performance), atau dengan mencapai penyelesaian atau [full combo](/wiki/Full_combo) pada beatmap dengan status [Qualified](/wiki/Beatmap/Category#qualified), [Ranked](/wiki/Beatmap/Category#ranked), atau [Approved](/wiki/Beatmap/Category#approved) pada rentang [star rating](/wiki/Beatmap/Difficulty#star-rating) tertentu.
```

File: wiki/Medals/id.md [hunk 3]
```diff
@@ -1,16 +1,59 @@
...
+
+Meskipun Monthly Beatmap Spotlights telah dihentikan sejak April 2018 untuk digantikan oleh [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018), seluruh medali yang bersangkutan masih dapat diperoleh hingga saat ini.
+
+## Seasonal Spotlights
+
+**Medali Seasonal Spotlights** adalah medali yang diberikan ketika pemain menyelesaikan beatmap pack yang dirilis sebagai bagian dari [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018). Agar medali dalam kategori ini dapat terbuka, pemain perlu bermain seluruh beatmap *pada pack mode game apapun* yang dipertandingkan dalam Seasonal Spotlight ranking chart. Penggunaan [mod pengurang kesulitan](/wiki/Game_modifier#difficulty-reduction) atau [mod peningkat kesulitan](/wiki/Game_modifier#difficulty-increase) diperbolehkan
```
Comment (mousesalt) [inline wiki/Medals/id.md:47]:
```suggestion
**Medali Seasonal Spotlights** adalah medali yang diberikan ketika pemain menyelesaikan beatmap pack yang dirilis sebagai bagian dari [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018). Agar medali dalam kategori ini dapat terbuka, pemain perlu bermain seluruh beatmap *pada pack mode game apapun* yang dipertandingkan dalam Seasonal Spotlight ranking chart. Penggunaan [mod pengurang kesulitan](/wiki/Game_modifier#difficulty-reduction) atau [mod peningkat kesulitan](/wiki/Game_modifier#difficulty-increase) diperbolehkan.
```

**Review by TPGPL — APPROVED** [review]

**Pull request merged**